subscribers:
  front_cam:
    channels: [ 'rgb','feat_0','feat_1']
    fusion: [ 'color','bayesian_inference','bayesian_inference']
    topic_name: '/elvation_mapping/pointcloud_semantic'
    semantic_segmentation: False
    segmentation_model: 'lraspp_mobilenet_v3_large' # detectron_coco_panoptic_fpn_R_101_3x, lraspp_mobilenet_v3_large
    show_label_legend: True
    data_type: pointcloud
    feature_config:
      name: 'DINO'
      interpolation: 'bilinear'
      model: "vit_small"
      patch_size:  16
      dim: 5
      dropout: False
      dino_feat_type: "feat"
      input_size: [80, 160]
      projection_type: "nonlinear"

    cam_info_topic: "/zed2i/zed_node/depth/camera_info"
    image_topic: "/zed2i/zed_node/left/image_rect_color"
    depth_topic: "/zed2i/zed_node/depth/depth_registered"
    cam_frame: "zed2i_right_camera_optical_frame"
    confidence: True
    confidence_topic: "/zed2i/zed_node/confidence/confidence_map"
    confidence_threshold: 10
    feature_extractor: False

  rgb_cam:
    fusion: [ 'image_color' ]
    topic_name_camera: "/zed2i/zed_node/left/image_rect_color"
    topic_name_camera_info: "/zed2i/zed_node/depth/camera_info"
    channels: [ "rgb_image" ]
    data_type: image

  grey_cam:
    fusion: [ 'image_exponential' ]
    topic_name_camera: "/zed2i/zed_node/left/image_rect_gray"
    topic_name_camera_info: "/zed2i/zed_node/depth/camera_info"
    channels: [ "gray_image" ]
    data_type: image